web/src/ui/nodes/properties/authorship/authorship.ts

```diff
@@ -89,32 +89,37 @@ export class StencilaAuthorship extends LitElement {
     const textOpacity = getProvenanceOpacity(this.mi as ProvenanceOpacityLevel)
     const tooltipStyle = apply([
       'absolute bottom-[calc(100%+0.5rem)] left-1/2 z-10',
+      'opacity-0',
       'group-hover:opacity-100',
       'w-32',
-      'opacity-0',
       'rounded',
       'p-2',
       'transition-all delay-200 duration-300',
       'text-white text-sm',
       'bg-black',
       'transform -translate-x-1/2',
       'pointer-events-none',
-      'after:content[""]',
+      'after:content-[""]',
       'after:absolute after:-bottom-1 after:left-1/2',
       'after:w-2 after:h-2',
       'after:bg-black',
       'after:transform after:-translate-x-1/2 after:rotate-45',
     ])
 
+    // find the current text color in rgb (remove all whitespace)
+    const computedColour = window
+      .getComputedStyle(this)
+      .getPropertyValue('color')
+      .replace(/\s/g, '')
+
     /*
       Do not change the formatting of this template,
       line breaks between tags will introduce 
       whitespace into the text in the document preview.
     */
     // prettier-ignore
     const htmlTemplate = html`<span
-          class="group text-black"
-          style="position: relative; --tw-text-opacity: ${textOpacity};"
```

Comment:
gggrrr, what a pain, will look into this

Reply:
@nokome can you provide me with a way of creating this behaviour :) is it only in the table cells? or are the paragraphs / lists also showing this

Reply:
On this branch:

1. `make -C vscode build`
2. In VSCode press F5 to launch extension
3. Create a new `temp.smd` file
4. Type `::: do @insert-filled-table cat breeds` and run it
5. Edit one of the cells in the suggested table
6. Expand the node card for the table

![image](https://github.com/stencila/stencila/assets/1152336/bc8ffc5d-95a2-4cfd-a537-284bae85876f)

Also, the tooltip is not wide enough for the content and does not user Inter as do the other tooltips:

[Screencast from 2024-06-05 12-08-28.webm](https://github.com/stencila/stencila/assets/1152336/e39aefc3-ac03-4a7f-8cc8-a5a2b2b4b597)

Reply:
hmm this is a strange one, i do get the line break when I first open the table card.
but if i close it and re-open it again it displays as expected...

https://github.com/stencila/stencila/assets/73506758/24dd2632-6de3-4205-8d38-9bd8314eaa3e

Reply:
I've just checked this on `main` (3a4e7b3f82a70949fc3b28f74e3989e97bd09ae0), following the above steps and can confirm it is NOT happening there (although the authorship tooltip width is not wide enough there too). So maybe the way to resolve this is examine/revert the diffs in this PR one-by-one until you get the previous, correct behavior.

Reply:
yes good idea, thanks for your patience :)

Reply:
Okay it seems that the problem here is the tooltip and how it renders initially.

If I take the tooltip out of the picture, it works properly (sans tooltip of course).

The tooltip in its current form was only ever a hack job to avoid using the `sl-tooltip` which was causing the problems before. I suggest we remove it for now, and I can create a more robust solution

![image](https://github.com/stencila/stencila/assets/73506758/734f997a-27b6-4be5-9717-1e33deb3a767)

Reply:
> I suggest we remove it for now, and I can create a more robust solution

Agreed.